app/src/main/java/unipd/se18/ocrcamera/ResultActivity.java

```diff
@@ -33,6 +44,8 @@
      */
     private TextView mOCRTextView;
 
```

Comment:
I was thinking to add an "explanation" like this:
```suggestion
     /**
      * The TextView of the extracted barcode from the captured photo.
      */
```

Reply:
Thanks for the suggestion! I'm fixing it!